SEED-platform/seed
# Activate "Save Changes" button for modified rules #2114

#### Any background context you want to provide?
"Save Changes" button was not activated correctly in several conditions.

#### What's this PR do?
Reactivate "Save Changes" button when:
1. Enabling or disabling rules;
2. Select all rules for enabling or disabling;
3. Add or delete a label.

#### How should this be manually tested?
Expect to see "Save Changes" button activated when:
1. Click checkboxes on the far left to enable or disable rules;
2. Create or remove a label;
3. Corner case: in "view by tax lot" tab, create or delete the label of last rule:
Before:
<img width="1276" alt="Screen Shot 2020-05-13 at 2 02 37 PM" src="https://user-images.githubusercontent.com/49968203/81859431-7a35e300-9522-11ea-8404-21099c1c546a.png">
<img width="1272" alt="Screen Shot 2020-05-13 at 2 02 46 PM" src="https://user-images.githubusercontent.com/49968203/81859438-7d30d380-9522-11ea-8cd4-e48cfe80f344.png">

After:
<img width="1267" alt="Screen Shot 2020-05-13 at 2 00 21 PM" src="https://user-images.githubusercontent.com/49968203/81859275-393dce80-9522-11ea-8bb5-1ef7af7a4f89.png">
<img width="1273" alt="Screen Shot 2020-05-13 at 2 00 28 PM" src="https://user-images.githubusercontent.com/49968203/81859270-380ca180-9522-11ea-86e4-3a9663222076.png">

#### What are the relevant tickets?
#2114 

#### Screenshots (if appropriate)
